joe/joe.py

```diff
@@ -88,25 +87,26 @@ def _fetch_gitignore(raw_name, directory=''):
     '''
     output = '\n#####=== %s ===#####\n' % raw_name
     if directory:
-        filepath = os.path.join(DATA_DIR, '%s' % directory + '/' + raw_name + \
-                                                 '.gitignore')
+        filepath = os.path.join(DATA_DIR, '%s/%s.gitignore' %
+                                (directory, raw_name))
     else:
         filepath = os.path.join(DATA_DIR, raw_name + '.gitignore')
         output += '\n'
     try:
-        with open(filepath) as f:
-            output += f.read()
+        with open(filepath) as gitignore:
```

Comment:
All examples of file IO use `f` as a standard (I'm assuming). https://docs.python.org/2/tutorial/inputoutput.html

I think it's best to leave it as `f`.